docs/layouts/partials/docs/footer.html

```diff
@@ -24,5 +24,8 @@
 {{ if eq "/zh" .Site.LanguagePrefix }}
 	{{ $folder = "content.zh" }}
 {{ end }}
+
+<a href="https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications">Want to contribute translation?</a>
+<br><br>
 <a href="{{ .Site.Params.Repo }}/edit/{{ .Site.Params.Branch }}/docs/{{ $folder }}/{{ .File.Path }}" style="color:black"><i class="fa fa-edit fa-fw"></i>Edit This Page</a>
```

Comment:
@sjwiesman I thought this might look nicer if it was on the same line.  Do you think it is worth the effort?  Is it just a matter of tweaking the .scss file?

Reply:
I think so, and yes that is all it should take.

Reply:
I know where it should be put but I gave up on this because...CSS

Reply:
thats fine